x/ibc/02-client/handler.go

```diff
@@ -104,3 +106,48 @@ func HandlerClientMisbehaviour(k keeper.Keeper) evidencetypes.Handler {
 		return nil
 	}
 }
+
+// HandleClientUpdateProposal is a handler for executing a passed client update proposal
+func HandleClientUpdateProposal(ctx sdk.Context, k keeper.Keeper, p *ibctypes.ClientUpdateProposal) error {
```

Comment:
nit, move content to `02-client/keeper/proposal.go` (i.e `func (k Keeper) ClientUpdateProposal(...)`)so that the only contents of this functions are the call to the `k.ClientUpdateProposal` function and events